cli-output/src/cli_output.rs

```diff
@@ -1634,6 +1634,42 @@ impl fmt::Display for CliSignOnlyData {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CliSignOnlyDatas {
+    pub cli_sign_only_datas: Vec<CliSignOnlyData>,
```

Comment:
If we're going to print more than one `CliSignOnlyData`, I think we need to add some context to be sure we can resolve them correctly